sdk/blockchain.go

```diff
@@ -37,28 +36,6 @@ func (b *BlockchainService) GetBlockByHeight(ctx context.Context, height Height)
 	return dto.toStruct()
 }
 
-// returns Transaction's inside of block at passed height
-func (b *BlockchainService) GetBlockTransactions(ctx context.Context, height Height) ([]Transaction, error) {
```

Comment:
Why did you remove this route?) We didn't remove it.

Reply:
@Cempl

Reply:
Sorry, my mistake. Rollback is done)